newIDE/app/src/EventsSheet/EventsTree/index.js

```diff
@@ -484,6 +495,23 @@ export default class ThemableEventsTree extends Component<
     }
   }
 
+  scrollToPosition(position: number) {
+    const currentList = this._list;
+    if (currentList) {
+      const listWrapper = currentList.wrappedInstance.current;
+      listWrapper && listWrapper.scrollToPosition(position);
+    }
+  }
```

Comment:
This is not actually used. Should I remove it?

Reply:
yes, which is not used either.

Reply:
Yes let's clean dead functions. Because these things are hard to get right, we might as well avoid making the API surface more complex than what it should be